src/test/java/teammates/test/driver/TestStringHelper.java

```diff
@@ -0,0 +1,23 @@
+package teammates.test.driver;
+
+import teammates.common.util.StringHelper;
+
+/**
+ * Holds additional methods for {@link StringHelper} used only in tests.
+ */
+public final class TestStringHelper {
```

Comment:
@damithc  @whipermr5  I  would also prefer the Option 3 so should we proceed with that?  And how suffix should be used ?  `StringHelperTestHelper` or `StringHelperExtension` which should be used  former look more descriptive and fine to me except somewhat long